backend/Makefile

```diff
@@ -34,6 +36,11 @@ testmigrateup:
 testmigratedown:
 	migrate -path ./migrations -database "postgres://postgres:admin@localhost:5432/flow_snapshot_test?sslmode=disable" -verbose down
 
+dev_run_migrate:
+	@echo Running migrations...
+	make --ignore-errors migrateup &>/dev/null
+	make --ignore-errors testmigrateup &>/dev/null
```

Comment:
I'm using `--ignore-errors` here because otherwise the output from migrate will show errors when there is no migrations to apply and I'm sending the output to null to prevent the bootup output from being noisy. Open to other suggestions on how we could do this.